devops/vaults.json

```diff
@@ -1,54 +0,0 @@
-[
```

Comment:
Can you explain how the keys from the vaults in 1Password get built into the env/app?

Reply:
![Clipboard - April 20, 2023 2_31 PM](https://user-images.githubusercontent.com/10002889/233495310-b6b82360-4b05-434d-953c-aef60aaf457c.png)